scripts/tasks/screener.ts

```diff
@@ -37,6 +35,8 @@ export async function screener() {
     if (!affectedPackages.has(affectedPackageInfo.packageJson.name)) {
       await cancelScreenerRun(screenerConfig, 'skipped');
     } else {
+      const screenerStates = await getScreenerStates(screenerConfig);
```

Comment:
_explainer_: The screener states can only be generated if the storybook exists, there's actually no need to do that until we're sure we want to run the screener tests

Reply:
can we add this intel to the source code  as well ?